README.md

```diff
@@ -13,13 +13,15 @@ Make sure you have the following installed:
 
```

Comment:
U ovoj sekciji ### Prerequisites treba li dodati onda PNPM i možda link na docs how to install?

https://pnpm.io/installation